src/utils.tsx

```diff
@@ -188,12 +205,16 @@ export const renderText = <Us extends DefaultUserType<Us> = DefaultUserType>(
 
     if (noParsingNeeded.length > 0 || linkIsInBlock) return;
 
-    const displayLink = type === 'email' ? value : value.replace(detectHttp, '');
+    try {
+      const displayLink = type === 'email' ? value : formatUrlForDisplay(href);
 
-    newText = newText.replace(
-      new RegExp(escapeRegExp(value), 'g'),
-      `[${displayLink}](${encodeURI(decodeURI(href))})`,
-    );
+      newText = newText.replace(
+        new RegExp(escapeRegExp(value), 'g'),
+        `[${displayLink}](${encodeDecode(href)})`,
+      );
+    } catch (e) {
+      console.log('meh');
```

Comment:
--> console.log('meh') <-- what is this wizardry

Reply:
Good catch